src/auth/UserRequest.h

```diff
@@ -247,7 +247,7 @@ int authenticateAuthUserRequestIPCount(Auth::UserRequest::Pointer);
 
 /// \ingroup AuthAPI
 /// See Auth::UserRequest::authenticated()
-int authenticateUserAuthenticated(Auth::UserRequest::Pointer);
+bool authenticateUserAuthenticated( Auth::UserRequest::Pointer);
```

Comment:
```suggestion
bool authenticateUserAuthenticated(const Auth::UserRequest::Pointer &);
```

Reply:
The const is unnecessary; it was there and I removed as per [Alex' review](https://github.com/squid-cache/squid/pull/1679#discussion_r1493464919)